### ISSUE TITLE
All emails get marked as read in inbox

### ISSUE THREAD

**Issue by axelsimon** [mailpile/Mailpile#1858]
Hi,
I was trying to use Mailpile (v.1.0.0rc0) in parallel to Thunderbird and saw that while emails looked unread in Mailpile (with unread showing up on top when sorting by "freshness"), the same emails were marked as read in Thunderbird, even though I'd never read them.

It also appears that all the emails that were still marked as unread in my inbox have now been marked as read by Mailpile.
Is this a know issue? Is this intended behaviour?

I see something similar was discussed in #1152 but I think this is different and specific enough to warrant its own issue.

Thanks.

**Comment by Kabouik** [comment]
I observed the same this morning. I am not sure whether the same behavior occurred with the previous version of Mailpile, but I don't think so.

This can be annoying because if literally makes users _have to_ use Mailpile to check their emails and not miss the unread ones. If you're just trying out Mailpile but are not sure yet you'll keep using it, or if you need to access emails from computers/smartphones where there is no Mailpile installation, then you cannot identify unread emails.

**Comment by BjarniRunar** [comment]
This depends on the IMAP/POP3 server. Some servers (GMail does this, if I recall) will mark a message as read, if it has been downloaded. That's wrong and horrible, and not exactly our fault.

But especially in the GMail case, they are so big and important that we should probably still attempt some sort of workaround to prevent the problem.

(As a reminder to people browsing this thread: Mailpile currently only downloads mail, it does not perform any synchronization. That means reading or deleting things in the Mailpile UI will not have any effect on what is seen in other mail clients, since the changes made within Mailpile are not synced back to the mail server. Similarly, if you delete/read/... on the server AFTER Mailpile has downloaded a message, the changes will not be propagated to Mailpile. Yes we know this annoys people, but it's a hard problem and not one we will be able to fix in the near term.)

**Comment by Kabouik** [comment]
Yandex does this too. I will check if this is the case with Openmailbox.org too (it's an open source email provider that respects privacy), but importation with of Openmailbox emails in Mailpile is not perfect yet anyway (draft are not found, sent emails are not displayed in chronological order).

About your remainder, I did know about that, but your message makes me wander if this is by design, or if true synchronization is the final objective for Mailpile?

**Comment by BjarniRunar** [comment]
@Kabouik - it is not exactly by design. We left that out deliberately because we recognized that it was complicated and we felt we could do without for a while. I haven't yet come up with a design for implementing bi-directional sync this that I am happy with.

**Comment by Kabouik** [comment]
Thanks for the clarification @BjarniRunar.

I can confirm that emails get marked as read with Openmailbox as well. However, I did use some clients before that did not mark emails as read on Yandex, Gmail or Openmailbox when downloading with IMAP (it was 2-way synchronization in that case and emails would just be marked as read when actually read, either on the webmail or the clients). How is that different? They were open source, in case the code may be of help.

**Comment by BjarniRunar** [comment]
Marking this as 1.0 - we should try and figure out how not to accidentally mark things as read. It's possible, we should do it.

**Comment by BjarniRunar** [comment]
This should be solved by The Great @Smari ... please reopen or file another issue if we're totally wrong. :-)
